code/modules/mob/living/carbon/monkey/combat.dm

```diff
@@ -123,8 +123,8 @@
 	if(enemies[L])
 		return TRUE
 
-	// target non-monkey mobs when aggressive, with a small probability of monkey v monkey
-	if(aggressive && (!istype(L, /mob/living/carbon/monkey/) || prob(MONKEY_AGGRESSIVE_MVM_PROB)))
+	// target non-jungle mobs when aggressive, with a small probability of monkey v monkey
+	if(aggressive && (!faction_check_mob(L, 0) || prob(MONKEY_AGGRESSIVE_MVM_PROB)))
```

Comment:
this is a boo 
```suggestion
	if(aggressive && (!faction_check_mob(L, FALSE) || prob(MONKEY_AGGRESSIVE_MVM_PROB)))
```

Reply:
Thanks, Cobby. I had a feeling something was up with the aggresion code, but I just couldn't put my finger on it.